src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRunCodegenNodeFactory.cs

```diff
@@ -110,6 +110,8 @@ private ISymbolNode CreateReadyToRunHelperCell(ReadyToRunHelper helperId)
             SignatureContext signatureContext, 
             bool isUnboxingStub = false)
         {
```

Comment:
Do we need still need the check here?

Reply:
Removed in 9th commit. I have verified by rerunning the R2R unit test that the methods dealing with Vector are still blocked from CPAOT compilation with the change.